packages/plugin-core/src/components/lookup/TabUtils.ts

```diff
@@ -0,0 +1,136 @@
+import _ from "lodash";
+import semver from "semver";
+import { Event, version as vscodeVersion, ViewColumn, window } from "vscode";
+
+export interface Tab {
```

Comment:
can you add comments to each of these interfaces saying that these are copied from VSCode, and that they should be removed after we decide to bump to 1.67 or greater?